doc/nrf/ug_radio_coex.rst

```diff
@@ -44,44 +41,48 @@ To avoid conflicts, check the protocol documentation to see if the protocol uses
-#. On nRF53 devices, you must also apply the same Kconfig options mentioned in steps 1 and 2 to the network core - see :ref:`ug_multi_image`.
+2. Select the specific Coexistence implementation.
+   See :ref:`ug_radio_cx_wifi_coex_impls` for details about the implementations currently supported.
+#. On the nRF5340, you must apply the settings to the Kconfig options mentioned in steps 1 and 2 also to the network core.
+   See :ref:`ug_multi_image`.
 
 .. _ug_radio_cx_hardware_description:
 
 Hardware description
 ********************
 
-The |NCS| provides a wrapper that configures WiFi Coexistence based on devicetree (DTS) and Kconfig information.
+The |NCS| provides a wrapper that configures Wi-Fi Coexistence based on devicetree (DTS) and Kconfig information.
 
-To enable CX support for any for the currently supported CX implementation, you must add an ``nrf_radio_coex`` node in the devicetree source file,
-as described below. The node can also be provided by the devicetree source file of the target board or by an overlay file. See :ref:`zephyr:dt-guide` for
-more information about the DTS data structure, and :ref:`zephyr:dt_vs_kconfig` for information about differences between DTS and Kconfig.
+To enable CX support for the currently supported CX implementation, you must add an ``nrf_radio_coex`` node in the devicetree source file.
+You can also provide the node using the devicetree source file of the target board or an overlay file.
+See :ref:`zephyr:dt-guide` for more information about the DTS data structure, and :ref:`zephyr:dt_vs_kconfig` for information about differences between DTS and Kconfig.
 
-For the currently supported CX implementation the ``nrf_radio_coex`` node has the ``compatible`` property set to
-``generic-radio-coex-three-wire``. Devicetree nodes compatible with ``generic-radio-coex-three-wire`` are suitable for a situation in which a 3 wire
-interface to the PTA is provided, ideally such as the one shown in the image below. However, the role of each of the pins is dependent on the Wifi
-Coexistence implementation.
+For the CX implementation currently supported, the ``nrf_radio_coex`` node has the ``compatible`` property set to ``generic-radio-coex-three-wire``.
+Devicetree nodes compatible with ``generic-radio-coex-three-wire`` can be used when a three-wire interface to the PTA is provided (like the one shown in the following image).
+However, the role of each of the pins is dependent on the Wi-Fi Coexistence implementation used.
 
-Note: When adding custom user implementation a different name then ``nrf_radio_coex`` can be used for the node. Also, a new device binding can be
-added and used as the ``compatible`` property if ``generic-radio-coex-three-wire`` is unsuitable.
+.. note::
+   When adding a custom user implementation, you must use a different name than ``nrf_radio_coex`` for the node.
```

Comment:
> When adding a custom user implementation, you must use a different name than ``nrf_radio_coex`` for the node.

That's not accurate @FrancescoSer. What we wanted to state in this part of the doc is that while with the supported implementations this name _must_ be used, when a user provides their own implementation they are free to use any other naming.

Reply:
Ok, rephrased